plists/DefaultBookmark.plist

```diff
@@ -560,6 +560,8 @@
 	<false/>
 	<key>Terminal Type</key>
 	<string>xterm</string>
+	<key>Answerback String</key>
```

Comment:
No need to specify this; the default will be taken from code. This file could probably go away with a little work, but it definitely doesn't need to grow.